src/eval.c

```diff
@@ -203,6 +203,29 @@ static struct vimvar
     {VV_NAME("versionlong",	 VAR_NUMBER), VV_RO},
 };
 
+/*
+ * A type for template literals.
+ * e.g. "${some_literal}", "$var".
+ */
+typedef struct {
+    /*
+     * Does this have a form "${some_literal}" ?
+     * (if this is FALSE, this have a form "$var".)
+     */
+    int is_embedded_literal;
+
+    /*
+     * "some_literal" or "var" or else.
+     */
+    char_u* expr;
```

Comment:
```suggestion
    char_u *expr;
```

`*` should be prepended before a variable. Same for other declarations.

Reply:
Thanks a lot!
I did `:%s/\v([a-zA-Z_]+)\* ([a-zA-Z_])/\1 *\2/gc` :D